.github/workflows/release.yml

```diff
@@ -53,6 +53,65 @@ jobs:
     outputs:
       ENSO_RELEASE_ID: ${{ steps.prepare.outputs.ENSO_RELEASE_ID }}
       ENSO_VERSION: ${{ steps.prepare.outputs.ENSO_VERSION }}
+  enso-build-ci-gen-job-deploy-runtime-linux-x86_64:
+    name: Upload Runtime to ECR (linux, x86_64)
+    needs:
+      - enso-build-ci-gen-draft-release-linux-x86_64
+      - enso-build-ci-gen-job-upload-backend-linux-x86_64
+    runs-on:
+      - self-hosted
+      - Linux
+    steps:
+      - if: startsWith(runner.name, 'GitHub Actions') || startsWith(runner.name, 'Hosted Agent')
+        name: Setup conda (GH runners only)
+        uses: s-weigand/setup-conda@v1.2.1
+        with:
+          update-conda: false
+          conda-channels: anaconda, conda-forge
+      - if: startsWith(runner.name, 'GitHub Actions') || startsWith(runner.name, 'Hosted Agent')
+        name: Installing wasm-pack
+        uses: jetli/wasm-pack-action@v0.4.0
+        with:
+          version: v0.10.2
```

Comment:
do we still need wasm?

Reply:
We do still have WASM parser, the setup routine is shared across the actions.